File: SPCode/CS/UI/Dialog/SPPreview.cs
using System.Text.RegularExpressions;
using System.Diagnostics;
using System;
using System.Drawing.Drawing2D;
using System.Windows.Forms;
using System.Collections;
using System.Drawing;
using Microsoft.VisualBasic;
using System.Data.SqlClient;
using System.Data;
using System.Collections.Generic;
using WeifenLuo.WinFormsUI;
using Microsoft.Win32;
using WeifenLuo;

namespace SoftLogik.Win
{
	namespace UI
	{
		public partial class SPPreview
		{
			
			
		}
		public sealed class SPDefaultPreview
		{
			private static SPPreview m_objSPPreview = new SPPreview();
			
			public static SPPreview SPPreview
			{
				get
				{
					return m_objSPPreview;
				}
			}
		}
	}
	
	
	
	
	
}

File: SPCode/UI/Dialog/SPPreview.vb
Namespace UI
    Public Class SPPreview

    End Class
    Public Module SPDefaultPreview
        Private m_objSPPreview As SPPreview = New SPPreview

        Public ReadOnly Property SPPreview() As SPPreview
            Get
                Return m_objSPPreview
            End Get
        End Property
    End Module
End Namespace




